test.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
🚫 **[phpcs]** <WordPress.Files.FileName.InvalidClassFileName> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class file names should be based on the class name with "class-" prepended. Expected class-testfile.php, but found test.php.